Tests: Remove `originalContent` from fixtures

dmsnell opened the pull request:
## Status
- Ready for review, but I'm double-checking to make sure we don't re-introduce this or other properties when re-generating the fixture data.
	- ✅ confirmed that when re-generating the fixture data we keep only those attributes we want to. confirmed by erasing them all and running the tests with the `GENERATE_MISSING_FIXTURES` `ENV` value set.

## Description

Previously we've been asserting in our full-content snapshot tests that
the `originalContent` property doesn't change, but it's faulty and going
away (see #38460).

By removing it from the test fixtures we can relax our constraint that
it exists and doesn't change, leaving future work easier to review and focus.

This is a generated property while loading the blocks and use of it leads
to content corruption and loss on certain invalid blocks and blocks with
inner blocks.

After this change we're still veryifying the full load and save flow
for the block implementations.

## Testing Instructions
As this is a test-only change please review the code and decisions going
into the removal of the property. If the test suite passes there is no
other testing that needs to be performed.

## Types of changes
- Relax the constraint on `originalContent` in our full-content snapshot tests.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/accessibility/ -->
- [x] I've tested my changes with keyboard and screen readers. <!-- Instructions: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/accessibility-testing.md -->
- [x] My code has proper inline documentation. <!-- Guidelines: https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/ -->
- [x] I've included developer documentation if appropriate. <!-- Handbook: https://developer.wordpress.org/block-editor/ -->
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). <!-- React Native mobile Gutenberg guidelines: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/code/native-mobile.md -->
- [x] I've updated related schemas if appropriate. <!-- Reference: https://github.com/WordPress/gutenberg/tree/trunk/schemas -->

gziolo reviewed (approved):
It requires rebasing, but it looks good overall. I don't think that `originalContent` was useful in the context of those tests anyway.

dmsnell committed:
Tests: Remove originalContent from fixtures

Previously we've been asserting in our full-content snapshot tests that
the `originalContent` property doesn't change, but it's faulty and going
away (see #38460).

By removing it from the test fixtures we can relax our constraint that
it exists and doesn't change, leaving future work easier to review and focus.

This is a generated property while loading the blocks and use of it leads
to content corruption and loss on certain invalid blocks and blocks with
inner blocks.

After this change we're still veryifying the full load and save flow
for the block implementations.

dmsnell committed:
Remove originalContent proeprty from test fixtures

dmsnell commented:
> It requires rebasing, but it looks good overall. I don't think that originalContent was useful in the context of those tests anyway.

Thanks @gziolo - I think we might end up having to rebase this multiple times, or in my case, I regenerated the entire thing since it was easier than a rebase.
